pkg/systemd/services.jsx

```diff
@@ -880,22 +881,24 @@ const ServicesPage = () => {
 
     /* Listen for permission changes for "Create timer" button */
     useEvent(superuser, "changed");
-    const { path, options } = usePageLocation();
+    // trigger re-renders when location changes (e.g. through changing filters)
+    usePageLocation();
 
+    const options = cockpit.location.options;
     const activeTab = options.type || 'service';
     const owner = options.owner || 'system';
     const setOwner = (owner) => cockpit.location.go(cockpit.location.path, { ...cockpit.location.options, owner });
 
     if (owner !== 'system' && owner !== 'user') {
-        console.warn("not a valid location: " + path);
+        console.warn("not a valid location: " + JSON.stringify(cockpit.location));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18690-20230423-064143-36147809-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)